src/components/InputTooltip.js

```diff
@@ -109,18 +109,43 @@ class InputTooltip extends Component {
         }
     }
 
+    splitText(textToSplit, query) {
```

Comment:
textToSplit comes from API
so it might not be text for some odd reason
I think we should add a check for it here

Reply:
why it might not be a text?